src/IBI.WZDx/Models/RoadEvents/CdsCurbZonesReference.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using IBI.WZDx.Equality;
+
+namespace IBI.WZDx.Models.RoadEvents;
+
+/// <summary>
+/// Describes specific curb zones that are impacted by a work zone via an external reference to the
+/// curb data specification's.
+/// </summary>
+/// <param name="CdsCurbZoneIds">A list of CDS Curb Zone Ids</param>
```

Comment:
Missing period and for consistency with the WZDx docs, "Ids" should be `<c>id</c>s.` to be more clear its referencing the `id` property on the curb zone. Also, [like the WZDx docs](https://github.com/usdot-jpo-ode/wzdx/blob/v4.2/spec-content/objects/CdsCurbZonesReference.md), please do a `see href` to the CDS Curb Zone.